IdokladSdk/Requests/Account/Subscription/Filter/SubscriptionFilter.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace IdokladSdk.Requests.Account.Subscription.Filter
+{
+    /// <summary>
+    /// Subscription filter.
+    /// </summary>
+    public class SubscriptionFilter
```

Comment:
To se mně nelíbí, ale je mi jasné, že nemáš jinou možnost říct, že se to nedá podle ničeho třídit. Bylo by dobré, kdyby jsme kromě té třídy BaseList ze které je zděděn ten SubscriptionList měli ještě nějakou, která nepodporuje filtrování a nevyžadovala by pak takovou prázdnou třídu, ale tohle je asi okrajový případ, takže to nebudeme řešit.